src/azure-cli/azure/cli/command_modules/acs/addonconfiguration.py

```diff
@@ -312,6 +312,8 @@ def ensure_container_insights_for_monitoring(
                 workspace_resource_id, "2015-11-01-preview"
             )
             location = resource.location
+            # location can have spaces for example 'East US' hence remove the spaces
+            location = location.replace(" ", "").lower()
```

Comment:
Could you refer to this method for the processing of location names?

https://github.com/Azure/azure-cli/blob/b6b5cbf4613ae22b6116a93fcf17042adc563abb/src/azure-cli-core/azure/cli/core/commands/parameters.py#L95-L100

Reply:
thanks.